### ISSUE TITLE
[MU4 Issue] [Accessibility] Master Palette window opens much slower than in previous builds when using Narrator

### ISSUE THREAD

**Issue by DmitryArefiev** [musescore/MuseScore#9880]
**Describe the bug**
Master Palette window opens much slower than in previous builds when using Narrator (now it takes 5-6sec instead of 2sec)
See video below

https://user-images.githubusercontent.com/90187801/143482062-e5aa1281-4c12-4a7c-a864-0dae50f484a3.mp4

**Expected behavior**

**Desktop:**
 - OS: Windows10, Narrator

**Comment by DmitryArefiev** [comment]
Tested on Narrator (Win10), Orca (Linux Mint 20.2) and VoiceOver (Mac12) - FIXED (should be in the next master)
